src/server.py

```diff
@@ -33,10 +26,6 @@ def listen_to(server):
         try:
             body, content_type = resolve_uri(parse_request(message))
             conn.sendall(response_ok() + content_type + b'\r\n\r\n' + body + b'\r\n\r\n')
```

Comment:
Could you update response_ok so that it takes the body and content type as arguments rather than having to concatenate them after the fact?